lib/core/src/model.rs

```diff
@@ -912,6 +926,9 @@ pub struct PaymentTxData {
     /// The tx ID of the transaction
     pub tx_id: String,
 
+    /// The destination address of the payment, only present when paying directly to a Liquid address
+    pub destination_address: Option<String>,
```

Comment:
IMO this is not clear enough.

- Does this work only for "pay to Liquid" type Send? All Receive payments are technically paying to a Liquid address, so they match the description.
- Does this also work for "Send to invoice" type Send, where we default to the MRH? In that case, we're also sending to a Liquid address.
- Does this also work for refund txs for failed Sends? In that case, the destination address is also a Liquid address.
- Since this is a struct that represents an onchain tx, what type of tx will this field be filled for? (lockup, claim, refund)

Please update the description accordingly.

Reply:
I've thought up a better way to go about this, as I agree that using the `PaymentTxData` struct is misleading for other payment types.